environment/src/main/java/io/github/kgress/scaffold/environment/config/DesiredCapabilitiesConfigurationProperties.java

```diff
@@ -167,6 +167,18 @@
      * Optional. The custom time zone to use when spinning up the sauce lab VM.
      */
     private String timeZone;
+
+    /**
+     * This enables the extended debugging feature on SauceLabs. The default on SauceLabs is to have this set as false.
+     * Required to be true for capturePerformance to be enabled.
+     */
+    private boolean extendedDebugging = false;
+
+    /**
+     * This enables the performance capturing feature on SauceLabs. The default on SauceLabs is to have this set as false.
+     * extendedDebugging must also be true for this to work.
+     */
+    private boolean capturePerformance = false;
```

Comment:
On usage with SauceLabs: if these values are just not present in the request, does SauceLabs automatically default it as `false`?

If that's the case, I think we should remove the default value being set here as `false`, and then update the `WebDriverManager` to build `Optional.ofNullable()` values, check if these values are present, and set them as sauce capabilities. That logic [is found here](https://github.com/kgress/scaffold/blob/551e6d151e4c22aeef980e5b7637dc4e0424b94f/framework/src/main/java/io/github/kgress/scaffold/WebDriverManager.java#L607). These two values would be set just under like 610.

Reply:
Saucelabs defaults the values to false. I've made the changes to not send these values if they are unset.